test/addons/at-exit/binding.cc

```diff
@@ -3,7 +3,11 @@
 #include <node.h>
 #include <v8.h>
 
+// TODO(addaleax): Maybe merge this file with the cctest for AtExit()?
```

Comment:
Do you want to keep this as a todo or should we open an issue instead?

Reply:
I can open an issue about it once this lands. :+1: